src/pages/Examples.spec.tsx

```diff
@@ -1,16 +1,94 @@
+import Block from '@dojo/framework/widget-core/meta/Block';
+import WidgetBase from '@dojo/framework/widget-core/WidgetBase';
 import harness from '@dojo/framework/testing/harness';
+import { Constructor, WidgetMetaConstructor, MetaBase } from '@dojo/framework/widget-core/interfaces';
 import { tsx } from '@dojo/framework/widget-core/tsx';
+import { w } from '@dojo/framework/widget-core/d';
+
+import FontAwesomeIcon from '../widgets/icon/FontAwesomeIcon';
+import Grid from '../widgets/grid/Grid';
+import Landing from '../widgets/Landing';
+import LinkedCard from '../widgets/card/LinkedCard';
+import Subsection from '../widgets/section/Subsection';
 
 import Examples from './Examples';
 import * as css from './Examples.m.css';
 
+const mockMetaMixin = <T extends Constructor<WidgetBase<any>>>(Base: T, mockStub: jest.Mock): T => {
```

Comment:
We should think about optimizing `Block`-based testing at some point. For now, `Block`-powered widgets each mock their own meta, which works, but we could probably unify this into a nice helper API.